app/components/dataPanel.tsx

```
@@ -30,20 +35,16 @@ const excludedProperties = [
 
 export default function DataPanel({ obj, setObj, url }: Props) {
 
-    const containerRef = useRef<HTMLDivElement>(null);
-    const [containerHeight, setContainerHeight] = useState(0);
-
-    useEffect(() => {
-        if (containerRef.current) {
-            setContainerHeight(containerRef.current.clientHeight);
-        }
-    }, [containerRef.current]);
+    debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)